lib/tags/changelog.js

```diff
@@ -18,8 +32,9 @@ module.exports = function changelog (hexo) {
   const pathToChangelogs = path.resolve('source', '_changelogs', '*')
 
   return globAsync(pathToChangelogs)
-  .call('reverse')
   .then((paths) => {
+    paths.sort(versionNumberSort)
```

Comment:
javascript's mutation sort is usually frowned on... prob better to use `_.sortBy`

Reply:
`_.sortBy` doesn't let you use an arbitrary function to sort by, but I did update the PR to do `Array.sort` into a new array